framework/generated/generated_vulkan_struct_to_json.cpp

```diff
@@ -11130,6 +11142,8 @@ void FieldToJson(nlohmann::ordered_json& jdata, const Decoded_VkDeviceFaultVendo
         FieldToJson(jdata["applicationNameOffset"], decoded_value.applicationNameOffset, options);
         FieldToJson(jdata["applicationVersion"], decoded_value.applicationVersion, options);
         FieldToJson(jdata["engineNameOffset"], decoded_value.engineNameOffset, options);
+        FieldToJson(jdata["engineVersion"], decoded_value.engineVersion, options);
```

Comment:
Maybe worth a note in the PR summary that if by some chance someone had a capture containing this struct before 1.3.250, decoding it will fail because the struct size and contents has changed.

Reply:
We should also note it on new version release highlight.